js/App.js

```diff
@@ -125,11 +127,33 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits) {
+  if (maxDigits === undefined) { // Allow maxDigits = 0
```

Comment:
Defaulting an argument is a very common practice in JS, but it's more concisely done as:

```
maxDigits = maxDigits || 8;
```